app/_src/gateway/admin-api/index.md

```diff
@@ -2950,6 +2950,27 @@ has been configured on, the higher its priority.
 The complete order of precedence when a plugin has been configured multiple
 times is:
 
+{% if_version gte:3.4.x %}
+
+1. Plugins configured on a combination of: a Consumer, a Route, and a Service.
```

Comment:
See my previous suggested copy edit for this numbered list

Reply:
@Guaris This one still needs updating.